src/components/CookieConsent.tsx

```diff
@@ -0,0 +1,73 @@
+import { posthog } from "posthog-js";
+import { useEffect, useState } from "react";
+
+const CookieConsent = () => {
```

Comment:
I'm not seeing the cookie consent banner show up, but I am seeing event data going to PostHog.